webpack.config.js

```diff
@@ -3,7 +3,8 @@ var path = require("path");
 
 var entries = {
   background: "./webextension/background.js",
-  manage: "./webextension/manage.js"
+  manage: "./webextension/manage.js",
+  "babel-polyfill"
```

Comment:
We need the multiple entries because there are different contexts (background script vs. editor).  Am very curious how this plays out.